server/app/controllers/ti/TrustedIntermediaryController.java

```diff
@@ -129,17 +131,30 @@ public Result addApplicant(Long id, Http.Request request) {
               + trustedIntermediaryUrl,
           form);
     }
+    return redirectToDashboardWithError(
+        getFormErrors(tiClientCreationResult.getForm().get()),
+        tiClientCreationResult.getForm().get());
+  }
+
+  private String getFormErrors(Form<AddApplicantToTrustedIntermediaryGroupForm> form) {
+    StringBuilder errorMessage = new StringBuilder();
+    form.errors().stream()
+        .forEach(
+            validationError ->
+                errorMessage.append(
+                    System.getProperty("line.separator") + validationError.message()));
```

Comment:
question (blocking): why use `System.getProperty("line.separator")` here? the server's operating system shouldn't have any impact on what we display in the browser

Reply:
When multiple errors are present in a form, all the errors were concatenated and shown to the end user. After discussing with @clouser2  - we decided we would add a line break.

Reply:
Ah, you can just use `"\n"` instead

Reply:
Done!